src/Secret.php

```diff
@@ -0,0 +1,94 @@
+<?php
+
+namespace ItkDev\AzureKeyVault;
+
+/**
+ * Class Secret.
+ */
+class Secret
+{
+    private $id;
+    private $managed;
+    private $value;
+    private $enabled;
+    private $created;
+    private $updated;
+    private $recoveryLevel;
+
+    public function __construct($id, $value, $managed, $enabled, $created, $updated, $recoveryLevel)
```

Comment:
Missing function description, parameter and return also missing types?

For all function is this class.